src/main/scala/org/clulab/habitus/actions/HabitusActions.scala

```diff
@@ -170,5 +170,45 @@ class HabitusActions extends Actions {
       )
     }
   }
+  def splitIntoBinary(mentions: Seq[Mention]): Seq[Mention] = {
+    val toReturn = new ArrayBuffer[Mention]()
+    val (target, other) = mentions.partition( m => {
+      val valueLabels = m.arguments("value").map(_.label)
+      valueLabels.length > 1 &&
+      valueLabels.distinct.length == 1
+    }
+    )
+    for (m <- target) {
+      for (v <- m.arguments("value")) {
+        val newArgs = Map("variable" -> m.arguments("variable"), "value" -> Seq(v))
+        val newMention = copyWithArgs(m, newArgs = newArgs)
+        toReturn.append(newMention)
+      }
+    }
+    toReturn ++ other
+  }
+
+  def listStringOverlap(string: String, list: Seq[String]): Boolean = {
+    list.contains(string)
+  }
+
+  def measurementIsAppropriate(m: Mention): Boolean = {
+    // check if any of the possible units shows up in the norm
+    val probablyUnit = m.norms.get.head.split("\\s").last // e.g., get "m2" from "6 m2"; assume value is separated from unit with a space and unit is one token
+    val appropriateUnits = m.label match {
+      case "Quantity" => Seq("t/ha", "kg/ha", "kg", "d", "cm") // fixme: added d and cm for now to make sure tests pass, but do we want days and cm to be quantities?
+      case "AreaSize" => Seq("ha")
+      case _ => ???
```

Comment:
It's trivial, but could a runtime exception be thrown here with a message, like "Unknown measurement label ${m.label}"?  Right now it is clear from the rules that it can never happen, but later it might not be and it would be handy to see the culprit label which isn't displayed for a NotImplementedError thrown by ???.  Someone could fix it then without needing to add a println or start a debugger.  Some places probably have a lint rule for that.